diff -pruN jbigkit-2.1.o/libjbig/jbig.c jbigkit-2.1/libjbig/jbig.c
--- jbigkit-2.1.o/libjbig/jbig.c	2014-03-27 21:47:15.000000000 +0300
+++ jbigkit-2.1/libjbig/jbig.c	2020-09-18 18:42:09.605768600 +0300
@@ -38,6 +38,7 @@
 #include <assert.h>
 
 #include "jbig.h"
+#include "jbig_ar.h"
 
 #define MX_MAX  127    /* maximal supported mx offset for
 			* adaptive template in the encoder */
diff -pruN jbigkit-2.1.o/libjbig/jbig.h jbigkit-2.1/libjbig/jbig.h
--- jbigkit-2.1.o/libjbig/jbig.h	2014-03-27 21:47:15.000000000 +0300
+++ jbigkit-2.1/libjbig/jbig.h	2020-09-18 18:41:23.277557000 +0300
@@ -8,7 +8,10 @@
 #define JBG_H
 
 #include <stddef.h>
-#include "jbig_ar.h"
+#include "jbig-export.h"
+
+struct jbg_ardec_state;
+struct jbg_arenc_state;
 
 /*
  * JBIG-KIT version number
@@ -191,43 +194,43 @@ struct jbg_dec_state {
 
 /* function prototypes */
 
-void jbg_enc_init(struct jbg_enc_state *s, unsigned long x, unsigned long y,
+JBIG_EXPORT void jbg_enc_init(struct jbg_enc_state *s, unsigned long x, unsigned long y,
 		  int planes, unsigned char **p,
 		  void (*data_out)(unsigned char *start, size_t len,
 				   void *file),
 		  void *file);
-int jbg_enc_lrlmax(struct jbg_enc_state *s, unsigned long mwidth,
+JBIG_EXPORT int jbg_enc_lrlmax(struct jbg_enc_state *s, unsigned long mwidth,
 		   unsigned long mheight);
-void jbg_enc_layers(struct jbg_enc_state *s, int d);
-int  jbg_enc_lrange(struct jbg_enc_state *s, int dl, int dh);
-void jbg_enc_options(struct jbg_enc_state *s, int order, int options,
+JBIG_EXPORT void jbg_enc_layers(struct jbg_enc_state *s, int d);
+JBIG_EXPORT int  jbg_enc_lrange(struct jbg_enc_state *s, int dl, int dh);
+JBIG_EXPORT void jbg_enc_options(struct jbg_enc_state *s, int order, int options,
 		     unsigned long l0, int mx, int my);
-void jbg_enc_out(struct jbg_enc_state *s);
-void jbg_enc_free(struct jbg_enc_state *s);
+JBIG_EXPORT void jbg_enc_out(struct jbg_enc_state *s);
+JBIG_EXPORT void jbg_enc_free(struct jbg_enc_state *s);
 
-void jbg_dec_init(struct jbg_dec_state *s);
-void jbg_dec_maxsize(struct jbg_dec_state *s, unsigned long xmax,
+JBIG_EXPORT void jbg_dec_init(struct jbg_dec_state *s);
+JBIG_EXPORT void jbg_dec_maxsize(struct jbg_dec_state *s, unsigned long xmax,
 		     unsigned long ymax);
-int  jbg_dec_in(struct jbg_dec_state *s, unsigned char *data, size_t len,
+JBIG_EXPORT int  jbg_dec_in(struct jbg_dec_state *s, unsigned char *data, size_t len,
 		size_t *cnt);
-unsigned long jbg_dec_getwidth(const struct jbg_dec_state *s);
-unsigned long jbg_dec_getheight(const struct jbg_dec_state *s);
-unsigned char *jbg_dec_getimage(const struct jbg_dec_state *s, int plane);
-unsigned long jbg_dec_getsize(const struct jbg_dec_state *s);
-void jbg_dec_merge_planes(const struct jbg_dec_state *s, int use_graycode,
+JBIG_EXPORT unsigned long jbg_dec_getwidth(const struct jbg_dec_state *s);
+JBIG_EXPORT unsigned long jbg_dec_getheight(const struct jbg_dec_state *s);
+JBIG_EXPORT unsigned char *jbg_dec_getimage(const struct jbg_dec_state *s, int plane);
+JBIG_EXPORT unsigned long jbg_dec_getsize(const struct jbg_dec_state *s);
+JBIG_EXPORT void jbg_dec_merge_planes(const struct jbg_dec_state *s, int use_graycode,
 			  void (*data_out)(unsigned char *start, size_t len,
 					   void *file), void *file);
-unsigned long jbg_dec_getsize_merged(const struct jbg_dec_state *s);
-void jbg_dec_free(struct jbg_dec_state *s);
+JBIG_EXPORT unsigned long jbg_dec_getsize_merged(const struct jbg_dec_state *s);
+JBIG_EXPORT void jbg_dec_free(struct jbg_dec_state *s);
 
-const char *jbg_strerror(int errnum);
-void jbg_int2dppriv(unsigned char *dptable, const char *internal);
-void jbg_dppriv2int(char *internal, const unsigned char *dptable);
-unsigned long jbg_ceil_half(unsigned long x, int n);
-void jbg_split_planes(unsigned long x, unsigned long y, int has_planes,
+JBIG_EXPORT const char *jbg_strerror(int errnum);
+JBIG_EXPORT void jbg_int2dppriv(unsigned char *dptable, const char *internal);
+JBIG_EXPORT void jbg_dppriv2int(char *internal, const unsigned char *dptable);
+JBIG_EXPORT unsigned long jbg_ceil_half(unsigned long x, int n);
+JBIG_EXPORT void jbg_split_planes(unsigned long x, unsigned long y, int has_planes,
 		      int encode_planes,
 		      const unsigned char *src, unsigned char **dest,
 		      int use_graycode);
-int jbg_newlen(unsigned char *bie, size_t len);
+JBIG_EXPORT int jbg_newlen(unsigned char *bie, size_t len);
 
 #endif /* JBG_H */
diff -pruN jbigkit-2.1.o/libjbig/jbig_ar.h jbigkit-2.1/libjbig/jbig_ar.h
--- jbigkit-2.1.o/libjbig/jbig_ar.h	2014-03-27 21:47:15.000000000 +0300
+++ jbigkit-2.1/libjbig/jbig_ar.h	2020-09-18 18:41:04.684771500 +0300
@@ -8,6 +8,8 @@
 #ifndef JBG_AR_H
 #define JBG_AR_H
 
+#include "jbig-export.h"
+
 /*
  * Status of arithmetic encoder
  */
@@ -44,10 +46,10 @@ struct jbg_ardec_state {
 			 * place */
 };
 
-void arith_encode_init(struct jbg_arenc_state *s, int reuse_st);
-void arith_encode_flush(struct jbg_arenc_state *s);
-void arith_encode(struct jbg_arenc_state *s, int cx, int pix);
-void arith_decode_init(struct jbg_ardec_state *s, int reuse_st);
-int  arith_decode(struct jbg_ardec_state *s, int cx);
+JBIG_HIDDEN void arith_encode_init(struct jbg_arenc_state *s, int reuse_st);
+JBIG_HIDDEN void arith_encode_flush(struct jbg_arenc_state *s);
+JBIG_HIDDEN void arith_encode(struct jbg_arenc_state *s, int cx, int pix);
+JBIG_HIDDEN void arith_decode_init(struct jbg_ardec_state *s, int reuse_st);
+JBIG_HIDDEN int  arith_decode(struct jbg_ardec_state *s, int cx);
 
 #endif /* JBG_AR_H */
