--- CMakeLists.txt.o	2019-04-14 21:10:32.000000000 +0300
+++ CMakeLists.txt	2021-10-15 12:17:35.046893000 +0300
@@ -200,7 +200,7 @@ endif(PNG_HARDWARE_OPTIMIZATIONS)
 set(PNG_LIB_NAME png${PNGLIB_MAJOR}${PNGLIB_MINOR})
 
 # to distinguish between debug and release lib
-set(CMAKE_DEBUG_POSTFIX "d")
+#set(CMAKE_DEBUG_POSTFIX "d")
 
 include(CheckCSourceCompiles)
 option(ld-version-script "Enable linker version script" ON)
