--- CMakeLists.txt.o	2021-06-25 20:35:43.000000000 +0300
+++ CMakeLists.txt	2021-07-17 21:05:45.576875500 +0300
@@ -999,7 +999,7 @@ endforeach()
 if(WIN32)
     # Shared library
     if(NOT DEFINED BUILD_SHARED_LIBS OR BUILD_SHARED_LIBS)
-        set_target_properties(zlib PROPERTIES OUTPUT_NAME zlib${SUFFIX})
+        set_target_properties(zlib PROPERTIES OUTPUT_NAME z-sail)
     endif()
     # Static library
     if(NOT DEFINED BUILD_SHARED_LIBS)
@@ -1017,7 +1017,7 @@ if(WIN32)
     endif()
 else()
     # On unix-like platforms the library is almost always called libz
-    set_target_properties(${ZLIB_INSTALL_LIBRARIES} PROPERTIES OUTPUT_NAME z${SUFFIX})
+    set_target_properties(${ZLIB_INSTALL_LIBRARIES} PROPERTIES OUTPUT_NAME z-sail)
 endif()
 
 if(NOT DEFINED BUILD_SHARED_LIBS OR BUILD_SHARED_LIBS)
@@ -1060,7 +1060,7 @@ if(NOT DEFINED BUILD_SHARED_LIBS OR BUIL
     elseif(WIN32)
         # Creates zlib1.dll when building shared library version
         if(ZLIB_COMPAT)
-            set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
+            #set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
         else()
             set_target_properties(zlib PROPERTIES SUFFIX "2.dll")
         endif()
