--- CMakeLists.txt.o	2019-12-31 10:10:30.000000000 +0300
+++ CMakeLists.txt	2020-09-17 20:23:06.270026500 +0300
@@ -56,6 +56,7 @@ if(MSVC)
     RUNTIME_OUTPUT_NAME jpeg${SO_MAJOR_VERSION})
   # The jsimd_*.c file is built using /MT, so this prevents a linker warning.
   set_target_properties(jpeg PROPERTIES LINK_FLAGS "/NODEFAULTLIB:LIBCMT /NODEFAULTLIB:LIBCMTD")
+  set_target_properties(jpeg PROPERTIES RUNTIME_OUTPUT_NAME jpeg${SO_MAJOR_VERSION}-sail OUTPUT_NAME jpeg-sail)
 elseif(MINGW)
   set_target_properties(jpeg PROPERTIES SUFFIX -${SO_MAJOR_VERSION}.dll)
 endif()
