--- CMakeLists.txt.o	2021-04-15 20:37:29.440127000 +0300
+++ CMakeLists.txt	2021-04-15 20:38:27.980208900 +0300
@@ -403,6 +403,7 @@ set_target_properties(avif
                           VERSION ${LIBRARY_VERSION}
                           SOVERSION ${LIBRARY_SOVERSION}
                           C_VISIBILITY_PRESET hidden)
+set_target_properties(avif PROPERTIES OUTPUT_NAME sail-extra-avif)
 target_compile_definitions(avif
                            PRIVATE ${AVIF_PLATFORM_DEFINITIONS} ${AVIF_CODEC_DEFINITIONS})
 target_link_libraries(avif
