--- CMakeLists.txt.o	2024-07-30 22:15:08.000000000 +0300
+++ CMakeLists.txt	2025-10-09 17:23:26.306970096 +0300
@@ -609,6 +609,7 @@ endif()
 
 # Main avif library.
 set_target_properties(avif PROPERTIES VERSION ${LIBRARY_VERSION} SOVERSION ${LIBRARY_SOVERSION})
+set_target_properties(avif PROPERTIES OUTPUT_NAME sail-extra-avif)
 target_link_libraries(avif PRIVATE avif_obj)
 target_include_directories(avif PUBLIC $<BUILD_INTERFACE:${libavif_SOURCE_DIR}/include> $<INSTALL_INTERFACE:include>)
 if(BUILD_SHARED_LIBS)
